### PR TITLE
Use core rule config constants (alpha)

### PR DESCRIPTION
**Pull request by thc202**
Replace literal string with corresponding core rule config constant.
Update changes in ZapAddOn.xml file and update minimum ZAP version,
required to use the constant.
Add TODO task to LinkTargetScanner to replace one rule config when
available.

### CODE REVIEW

File: src/org/zaproxy/zap/extension/ascanrulesAlpha/SQLInjectionMsSQL.java [hunk 1]
```diff
@@ -158,9 +159,9 @@ public void init() {
 
 		// Read the sleep value from the configs
 		try {
-			this.sleep = this.getConfig().getInt("rules.common.sleep", DEFAULT_SLEEP_TIME);
+			this.sleep = this.getConfig().getInt(RuleConfigParam.RULE_COMMON_SLEEP_TIME, DEFAULT_SLEEP_TIME);
 		} catch (ConversionException e) {
-			log.debug("Invalid value for 'rules.common.sleep': " + this.getConfig().getString("rules.common.sleep"));
+			log.debug("Invalid value for 'rules.common.sleep': " + this.getConfig().getString(RuleConfigParam.RULE_COMMON_SLEEP_TIME));
```
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/ascanrulesAlpha/SQLInjectionMsSQL.java]: Both occurrences?

**Comment by thc202** [comment]
The build failure is unrelated (a test is randomly failing in `OpenApiPre2dot0UnitTest`).

File: src/org/zaproxy/zap/extension/ascanrulesAlpha/SQLInjectionMsSQL.java (hunk 1 above)
Comment (thc202) [reply src/org/zaproxy/zap/extension/ascanrulesAlpha/SQLInjectionMsSQL.java]: I did not change that in the other classes, as you prefer.

Comment (kingthorin) [reply src/org/zaproxy/zap/extension/ascanrulesAlpha/SQLInjectionMsSQL.java]: That’s fine, stay consistent. Dunno why I didn’t notice it earlier.

Comment (thc202) [reply src/org/zaproxy/zap/extension/ascanrulesAlpha/SQLInjectionMsSQL.java]: OK.

**Comment by thc202** [comment]
The build is now passing.

**Pull request merged**